test/distrib/csharp/run_distrib_test.bat

```diff
@@ -0,0 +1,21 @@
+
```

Comment:
What's with a blank line at top-of-file? Should this file not have a license header like almost every other in the codebase?